tests/integration/long/test_policies.py

```diff
@@ -11,16 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import inspect
+import os
+import time
 import unittest
+from typing import Optional
+from unittest.mock import Mock
 
 from cassandra import ConsistencyLevel, Unavailable
-from cassandra.cluster import ExecutionProfile, EXEC_PROFILE_DEFAULT
+from cassandra.cluster import ExecutionProfile, EXEC_PROFILE_DEFAULT, Session
+from cassandra.policies import LimitedConcurrencyShardConnectionBackoffPolicy, ConstantReconnectionPolicy, \
+    ShardConnectionBackoffPolicy, NoDelayShardConnectionBackoffPolicy, _ScopeBucket, \
+    _NoDelayShardConnectionBackoffScheduler
+from cassandra.shard_info import _ShardingInfo
 
 from tests.integration import use_cluster, get_cluster, get_node, TestCluster
 
 
 def setup_module():
+    os.environ['SCYLLA_EXT_OPTS'] = "--smp 8"
```

Comment:
This gonna be problematic on github actions, I don't know if you'll enough resources...

even if it would flat fail, it might make those test in this module unstable.

Reply:
ohh, done.